cmd/serve.go

```diff
@@ -109,6 +109,8 @@ func init() {
 	RootCmd.Flags().BoolVar(&ldapUseInsecure, "use-insecure", false, "Disable LDAP TLS")
 
 	RootCmd.Flags().DurationVar(&tokenTtl, "token-ttl", 24*time.Hour, "TTL for the token")
+	RootCmd.Flags().StringVar(&keypairDir, "keypair-dir", "./", "directory that contains keypair for signing/verifying tokens. Defaults to current directory")
```

Comment:
This should default to a subdirectory so that it is easy to mount a volume underneath it. Change the Dockerfile to create this directory in the image.

Do we really need the directory to be configurable?

Reply:
I will change it to subdirectory. I would prefer to keep it configurable even if we use a fixed path for now

Reply:
done.